dataikuapi/dss/ml.py

```diff
@@ -2499,6 +2499,10 @@ def get_roc_curve_data(self):
 
         return roc
 
+    @property
+    def _full_model_like_id(self):
+        return self.details["fullModelId"]
+
```

Comment:
I would set that in the base class, as it should work for clustering models too.

Reply:
I'm not sure as drift won't work on a clustering model ? 

Or maybe, you wanted this method to be called by end users and in this case, we probably want to make it public ? WDYT ?

Reply:
Obviously, it will not be used for drift. :-)

Yes, I think it could be useful to users and we should make it public.

Reply:
Done